src/app/screens/LNURLAuth/index.tsx

```diff
@@ -1,140 +1,38 @@
-import Button from "@components/Button";
-import ConfirmOrCancel from "@components/ConfirmOrCancel";
-import Container from "@components/Container";
-import ContentMessage from "@components/ContentMessage";
-import PublisherCard from "@components/PublisherCard";
-import ResultCard from "@components/ResultCard";
-import React, { useState } from "react";
-import { useTranslation } from "react-i18next";
-import { useNavigate } from "react-router-dom";
-import ScreenHeader from "~/app/components/ScreenHeader";
-import { useNavigationState } from "~/app/hooks/useNavigationState";
-import { USER_REJECTED_ERROR } from "~/common/constants";
+import { useEffect, useState } from "react";
+import LNURLAuthComponent from "~/app/components/LNURLAuth";
+import Onboard from "~/app/components/onboard";
+import { useAccount } from "~/app/context/AccountContext";
+import { isAlbyOAuthAccount } from "~/app/utils";
 import api from "~/common/lib/api";
-import msg from "~/common/lib/msg";
-import type { LNURLAuthServiceResponse } from "~/types";
 
-function LNURLAuth() {
-  const { t } = useTranslation("translation", { keyPrefix: "lnurlauth" });
-  const { t: tCommon } = useTranslation("common");
-
-  const navigate = useNavigate();
-  const navState = useNavigationState();
-
-  const details = navState.args?.lnurlDetails as LNURLAuthServiceResponse;
-  const origin = navState.origin;
-
-  const [successMessage, setSuccessMessage] = useState("");
-  const [errorMessage, setErrorMessage] = useState("");
-  const [loading, setLoading] = useState(false);
-
-  async function confirm() {
-    try {
-      setLoading(true);
-      const response = await api.lnurlAuth({
-        origin,
-        lnurlDetails: details,
-      });
-
-      if (navState.isPrompt && origin?.host) {
-        const allowance = await api.getAllowance(origin.host);
-
-        if (allowance.lnurlAuth === false) {
-          await msg.request("updateAllowance", {
-            id: allowance.id,
-            lnurlAuth: true,
-          });
+export default function LNURLAuth() {
+  const { account } = useAccount();
+  const [hasMnemonic, setHasMnemonic] = useState(false);
+  const [albyOAuthAccount, setAlbyOAuthAccount] = useState(false);
+
+  useEffect(() => {
+    async function fetchAccountInfo() {
+      try {
+        const fetchedAccount = await api.getAccount();
+        const connectorType = isAlbyOAuthAccount(fetchedAccount.connectorType);
+        setAlbyOAuthAccount(connectorType);
```

Comment:
Maybe `setIsAlbyOAuthAccount(isAlbyOAuthAccount)`?

Reply:
we already have a function with the name isAlbyOAuthAccount. we cant use same name in the state variables isn't it?


variable name will be isAlbyOAuthAccount which is also a function name to determine connector type